src/display/font_loader.js

```diff
@@ -47,6 +47,13 @@ class FontLoader {
     this._document.fonts.add(nativeFontFace);
   }
 
+  removeNativeFontFace(nativeFontFace) {
+    this.nativeFontFaces = this.nativeFontFaces.filter(
+      x => x !== nativeFontFace
+    );
```

Comment:
Could we possibly change `this.nativeFontFaces` into a `Set` instead, to avoid having to do this?